lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -51,6 +51,33 @@ def check_export_status(url, timeout: 180)
         assert_status_reponse_required_field(response_body)
 
         @output = response_body['output']
+
+        assert_output_has_type_url(@output)
+      end
+
+      def assert_output_has_type_url(output)
+        output.each do |file|
+          ['type', 'url'].each do |key|
+            assert file.key?(key), "Output file did not contain \"#{key}\" as required"
+          end
+        end
+      end
+
+      def check_file_request(output)
+        headers = { accept: 'application/fhir+ndjson' }
+        output.each do |file|
```

Comment:
This method or the test that calls it needs to protect against `output` being nil, since if the previous test fails it might not be set.